recipes/geoutils/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "geoutils" %}
+{% set version = "v0.0.1" %}
+
+{% set github_url = "https://github.com/GlacioHack/GeoUtils" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+source:
+        url: "{{ github_url }}/archive/refs/tags/{{ version }}.tar.gz"
+        sha256: "7efde1a137978754d847370b2fbd7b458bd404cad3c265f4fc9ba4fba8891cfd"
+
+
+build:
+        number: 0
+        script: python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```

Comment:
better to use `{{ PYTHON }}` instead of `python` to make sure you're using the right python

Reply:
Fixed in 40a7623